src/analysis/retail/priest/holy/modules/talents/Archon/EmpoweredSurgesHoly.tsx

```diff
@@ -55,9 +61,20 @@ class EmpoweredSurgesHoly extends Analyzer {
         position={STATISTIC_ORDER.OPTIONAL(99)}
         size="flexible"
         category={STATISTIC_CATEGORY.HERO_TALENTS}
+        tooltip={
+          <>
+            {' '}
+            Breakdown: <br />
+            <SpellLink spell={TALENTS_PRIEST.EMPOWERED_SURGES_TALENT} />:{' '}
+            <ItemPercentHealingDone amount={this.empoweredSurgesHealing}></ItemPercentHealingDone>{' '}
+            <br />
+            <SpellLink spell={SPELLS.ECHO_OF_LIGHT_MASTERY} />:{' '}
+            <ItemPercentHealingDone amount={this.eolContrib}></ItemPercentHealingDone> <br />
```

Comment:
Instead of using `<br>`, wrap them in `<div>` instead as it renders better on all devices.